web/libs/editor/src/tags/object/Video/HtxVideo.jsx

```diff
@@ -125,6 +125,7 @@ const VideoConfig = observer(({ item }) => {
       onSpeedChange={item.handleSpeed}
       loopTimelineRegion={item.loopTimelineRegion}
       onLoopTimelineRegionChange={item.setLoopTimelineRegion}
+      minSpeed={Number(item.minplaybackspeed) || 1}
```

Comment:
we already did all constraints
```suggestion
      minSpeed={item.minplaybackspeed}
```